src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -329,6 +340,26 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
   OnUretprobes(event_timestamp, event_data.pid, event_data.tid, event_data.rax);
 }
 
+void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
+                                    const UserSpaceFunctionEntryPerfEventData& event_data) {
+  function_call_manager_->ProcessFunctionEntry(event_data.tid, event_data.function_id,
```

Comment:
I am thinking whether we also just want to call OnUprobes here and below

Reply:
The reason for the `OnUprobes` and `OnUretprobes` methods is the logic about detecting duplicated uprobes (an to factor that code out since it's used more than once). But that doesn't apply to user space instrumentation, for which, furthermore, we don't have the CPU that we use in that logic.

Reply:
Ah, I missed the missing cpu. then it makes sense.